libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -800,3 +826,14 @@ void AP_MotorsMulticopter::convert_pwm_min_max_param(int16_t radio_min, int16_t
     _pwm_min.set_and_save(radio_min);
     _pwm_max.set_and_save(radio_max);
 }
+
+// 10Hz check whether motors are running after arming
+bool AP_MotorsMulticopter::check_failsafe()
+{
+    if (is_zero(_fs_rpm)) {
```

Comment:
arming check for ESC telem if FS_RPM > 0, fail to arm if no ESC telem available on your motor mask

Reply:
Fixed